lighthouse-cli/test/smokehouse/frontends/firehouse.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license Copyright 2020 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview Smoke test runner.
+ * Used to test integrations that run Lighthouse within a browser (i.e. LR)
+ * Supports skipping and modifiying expectations to match the environment.
+ */
+
+/* eslint-disable no-console */
+
+const smokeTests = require('../test-definitions/core-tests.js');
+const {runSmokehouse} = require('../smokehouse.js');
+
+/**
+ * @param {Smokehouse.FirehouseOptions} options
+ */
+async function runFirehouse(options) {
+  const {smokehouse, urlFilterRegex, skip, modify} = options;
```

Comment:
these all get set by the person consuming firehouse, correct?

Reply:
yup